### ISSUE TITLE
Symbol(Deno.internal.rid) is `undefined`

### ISSUE THREAD

**Issue by bombillazo** [denoland/deno#23115]
Version: Deno 1.42.0

After upgrading to 1.42.0 and running some local deno scripts, we get the following error:
```sh
error: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'Symbol(Deno.internal.rid)')
    at TTY.#read (ext:deno_node/internal_binding/stream_wrap.ts:220:45)
    at TTY.readStart (ext:deno_node/internal_binding/stream_wrap.ts:90:17)
    at _tryReadStart (node:net:276:30)
    at ReadStream._read (node:net:856:7)
    at ReadStream.Readable.read (ext:deno_node/_stream.mjs:2999:16)
    at ReadStream.read (node:net:789:34)
    at resume_ (ext:deno_node/_stream.mjs:3349:16)
    at Array.processTicksAndRejections (ext:deno_node/_next_tick.ts:33:15)
    at eventLoopTick (ext:core/01_core.js:165:29)
```

**Comment by dsherret** [comment]
Looks like https://github.com/denoland/deno/issues/23107 ?

**Comment by bartlomieju** [comment]
@bombillazo can you try with latest canary? (`deno upgrade --canary`)

**Comment by bombillazo** [comment]
This was fixed in 1.42.1, thanks!
